qutebrowser/test/config/test_configtypes.py

```diff
@@ -1377,14 +1377,28 @@ def test_validate_exists_not_abs(self, os_path):
 
     def test_validate_expanduser(self, os_path):
         """Test if validate expands the user correctly."""
+        os_path.expandvars.side_effect = lambda x: x.replace('$BAR',
+                                                             '/home/foo/bar')
```

Comment:
You can just use `os_path.expandvars.side_effect = lambda x: x` since that's not the thing being tested here anyways.